### PR TITLE
BAI-740 modify stepper to display vertically

### PR DESCRIPTION
**Pull request by m19671**

### CODE REVIEW

File: frontend/pages/beta/model/new/upload.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        return null
+      })
+    return null
+  }
+
+  return (
+    <Paper variant='outlined' sx={{ my: { xs: 3, md: 6 }, p: { xs: 2, md: 3 } }}>
+      <Grid container justifyContent='space-between' alignItems='center'>
+        <Box />
+        <SchemaSelector
```
Comment (ARADDCC002) [inline frontend/pages/beta/model/new/upload.tsx:198]: The beta pages will handle the schema selection on another page, so I'd recommend removing it. We should be able to remove all of the other schema related stuff from the page as well (such as the change/setting new schema functions). 

For now we can just fetch the minimal schema and use that to render the form for test purposes.

**Review by ARADDCC002 — CHANGES_REQUESTED** [review]

File: frontend/src/Form/FormBeta.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}) {
+  const [tab, setTab] = useState('designer')
+  const onTabChange = (_event: any, newValue: any) => {
+    setTab(newValue)
+  }
+
+  return (
+    <>
+      <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
+        <Tabs value={tab} onChange={onTabChange}>
```
[the remark this answers is not in the document]
Comment (m19671) [reply frontend/src/Form/FormBeta.tsx:29]: These will be removed with BAI-742

File: frontend/pages/beta/model/new/upload.tsx (hunk 1 above)
Comment (m19671) [reply frontend/pages/beta/model/new/upload.tsx:200]: Again, removed in BAI-742

File: frontend/pages/beta/model/new/upload.tsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+import LoadingBar from '../../../../src/common/LoadingBar'
+import { MinimalErrorWrapper } from '../../../../src/errors/ErrorWrapper'
+import MultipleErrorWrapper from '../../../../src/errors/MultipleErrorWrapper'
+import Form from '../../../../src/Form/FormBeta'
+import ModelExportAndSubmission from '../../../../src/Form/ModelExportAndSubmission'
+import { RenderButtonsInterface } from '../../../../src/Form/RenderButtons'
+import RenderFileTab, { fileTabComplete, RenderBasicFileTab } from '../../../../src/Form/RenderFileTab'
+import SchemaSelector from '../../../../src/Form/SchemaSelector'
+import SubmissionError from '../../../../src/Form/SubmissionError'
+import Wrapper from '../../../../src/Wrapper'
```
[the remark this answers is not in the document]
Comment (m19671) [reply frontend/pages/beta/model/new/upload.tsx:20]: For the temporary display I agree but will definitely change in the full implementation.

**Review by ARADDCC002 — APPROVED** [review]

**Pull request merged**